api-reference/_hidden/dxGanttContextMenu/items.md

```diff
@@ -9,3 +9,83 @@ inheritsType: dxGanttContextMenuItem
 Configures context menu item settings.
 
 ---
+
+The context menu contains a set of default commands: **'addTask'**, **'taskDetails'** and **'deleteTask'**. Use the [contextMenu](/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/contextMenu) option to recreate the context menu items.
```

Comment:
```suggestion
The context menu contains a set of default commands: **'addTask'**, **'taskDetails'**, and **'deleteTask'**. Use the [contextMenu](/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/contextMenu) option to recreate the context menu items.
```

Reply:
"recreate the context menu items" Why do you need to 'recreate' these items?

Reply:
Without using this option the context menu has a set of default items. When you use this option , it removes the default set of items and creates new.